README.md

```diff
@@ -39,10 +39,10 @@ This is **unofficial** android app for [taiga.io](https://www.taiga.io/) (agile
 * ... and other cool things
 
 ## Design
-Probably sucks. I'm not very good at designing UI, but I'm doing my best. So, if you have any suggestions, please tell me. I'll be really glad to receive any feedback.  
+Probably sucks. I'm not very good at designing UI, but I'm doing my best. So, if you have any suggestions, please tell me. I'll be really glad to receive any feedback.
 
 ## Download
-Check out app on [Google Play](https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github) or go to [Releases](https://github.com/EugeneTheDev/TaigaMobile/releases) page to download apk.
+Check out the app on [Google Play](https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github) or go to the [Releases page](https://github.com/EugeneTheDev/TaigaMobile/releases) to download the latest apk.
 
 ## Issues
-Please consider telling about problems or suggestions on [Issues](https://github.com/EugeneTheDev/TaigaMobile/issues) page.
+If you have any problems or suggestions, please post an [Issue](https://github.com/EugeneTheDev/TaigaMobile/issues).
```

Comment:
`please post an issue` - I think the better one is `submit an issue`

Reply:
done